### ISSUE TITLE
Cold O2 losing pressure randomly

### ISSUE THREAD

**Issue by leibniztg** [tgstation/-tg-station#15387]
1. Make 3 cans of cool O2
(this causes the can pressure to drop, not a bug I guess)
2. Pump 2 cans into the third to have one high pressure -200c can of O2
3. Fill an empty tank from the cannister, so far so good
4. Attempt to fill another empty tank
5. Pressure starts dropping in both the tank and canister
6. Canister and tank both lose pressure until they become empty even though they arent open to the air

I was standing next to them and I didnt feel any cold so it wasnt just a leak.. I guess?
But why would cold O2 leak anyway?

**Comment by tkdrg** [comment]
@duncathan @neersighted

**Comment by phil235** [comment]
Something similar happened to me when testing my fixes for cryo cells. I emptied half of an O2 canister in the air to reduce its pressure then closed the canister and hooked it up to the cryo pipes. Now I don't remember if the bug happened directly there or if it was after I disconnected/Reconnected more canisters to the pipe but the pressure in the pipes and that canister started dropping until it reached zero. The O2 wasn't cooled and the freezer off the whole time.
